test/apps/gups.c

```diff
@@ -412,12 +411,12 @@ SHMEMRandomAccess(void)
 
   if (MyProc == 0) {
     fprintf( outFile, "Running on %d processors\n", NumProcs);
-    fprintf( outFile, "Total Main table size = 2^" FSTR64 " = " FSTR64 " words\n",
+    fprintf( outFile, "Total Main table size = 2^%" PRIu64 " = %" PRIu64 " words\n",
```

Comment:
This requires C99, but it seems reasonable to support a standard old enough to drive.